examples/webgl_skinning_simple.html

```diff
@@ -10,7 +10,7 @@
 
 		<div id="info">
 			<a href="https://threejs.org" target="_blank" rel="noopener">three.js</a> webgl - simple skinning -
-			<a href="https://github.com/mrdoob/three.js/blob/master/examples/models/skinned/simple/simple.blend" target="_blank" rel="noopener">Blender File</a>
+			<a href="https://github.com/mrdoob/three.js/blob/master/examples/models/gltf/SimpleSkinning.gltf" target="_blank" rel="noopener">GLTF File</a>
```

Comment:
Um, in this case I think it's better if the entire `<a>` tag is deleted. The path to the glTF asset can be found in the source code below.